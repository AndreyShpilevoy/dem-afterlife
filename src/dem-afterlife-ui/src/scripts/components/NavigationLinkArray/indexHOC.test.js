/* eslint no-undef: 0, fp/no-unused-expression: 0, fp/no-nil: 0, fp/no-mutation:0 */

import React from 'react';
import {shallow} from 'enzyme';
import * as ThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet';
import NavigationLinkArray from './index';

ThemedStyleSheet.default.get = () => {};

describe('NavigationLinkArray HOC', () => {
    it('match expected snapshot', () => {
        expect(shallow(<NavigationLinkArray theme={'default'} navigationLinkArray={[]}/>)).toMatchSnapshot();
    });
});
