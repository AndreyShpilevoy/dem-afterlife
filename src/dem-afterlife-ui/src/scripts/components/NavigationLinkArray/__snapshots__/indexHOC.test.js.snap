// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLinkArray HOC match expected snapshot 1`] = `
<NavigationLinkArrayPure
  navigationLinkArray={Array []}
  styles={
    Object {
      "list": "list-0-12",
      "separator": "separator-0-13",
    }
  }
/>
`;
